modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -626,7 +630,7 @@ AUI.add(
 
 				var value = [];
 
-				if (condition && instance._isBinaryCondition(index)) {
+				if (condition && instance._isBinaryCondition(index) && condition.operands[1]) {
```

Comment:
Hey @alinedoleron just for a better visualization, is a forms pattern group the if declarations with parentheses, so can you group these declarations? ex: `if((condition) && (instance._isBinaryCondition(index)) && (visible))`